.circleci/config.yml

```diff
@@ -822,7 +822,7 @@ jobs:
           name: Install test dependencies
           command: |
             apt-get update
-            apt-get install -qqy --no-install-recommends nodejs npm cvc4
+            apt-get install -qqy --no-install-recommends nodejs npm
```

Comment:
Do you still want to remove cvc4 after removing the test that depended on it not being there (https://github.com/ethereum/solidity/pull/11421#discussion_r671456289)? I mean, do you generally not need it or was it removed just because of that test?

Reply:
Yea we generally don't need it, so just thought it makes it lighter

Reply:
OK. Fine then.